### PR TITLE
[REVIEW] Add Force Atlas 2 to benchmarks

### PR DESCRIPTION
**Pull request by hlinsen**

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by hlinsen** [comment]
rerun tests

File: benchmarks/bench_algos.py [hunk 1]
```diff
@@ -177,6 +177,11 @@ def bench_bfs(gpubenchmark, anyGraphWithAdjListComputed):
     gpubenchmark(cugraph.bfs, anyGraphWithAdjListComputed, 0)
 
 
+def bench_force_atlas2(gpubenchmark, anyGraphWithAdjListComputed):
+    gpubenchmark(cugraph.force_atlas2, anyGraphWithAdjListComputed,
+                 max_iter=50)
```
Comment (afender) [inline benchmarks/bench_algos.py:182]: just curious why `max_iter=50` and not rely on the default 500 (or set default to 50)?

Comment (hlinsen) [reply benchmarks/bench_algos.py:182]: The benchmark takes a lot of time with 500 iterations. I wasn't sure if we care to have an algorithm that takes much more time to benchmark, so I wanted to keep it a minimum consistent with the other iterative algorithms.

**Pull request merged**